axolotl-web/src/pages/Contacts.vue

```diff
@@ -76,12 +76,12 @@
                 alt="Avatar image"
                 @error="onImageError($event)"
               />
-              {{ c.Name[0] ? c.Name[0] + c.Name[1] : c.Name[0] }}
+              {{ c.name[0] ? c.name[0] + c.name[1] : c.name[0] }}
             </div>
           </div>
           <div class="meta col-8" @click="contactClick(c)">
-            <p class="name">{{ c.Name }}</p>
-            <p class="number">{{ c.Tel }}</p>
+            <p class="name">{{ c.name }}</p>
+            <p class="number">{{ `+${ c.address.phonenumber.code.value} ${ c.address.phonenumber.national.value}` }}</p>
```

Comment:
This nullpointers if the phonenumber is not known. And its logic within the template. I want to suggest to introduce a function `getPhonenumber()` or similar.